app/cypress/integration/cif/project-revision/index.spec.js

```diff
@@ -125,16 +144,14 @@ describe("the new project page", () => {
       component: "Project Contacts Form",
       variant: "with errors",
     });
-    cy.findByText(/Quarterly reports/i).click();
-    cy.findByText(/Add quarterly reports/i).click();
-    cy.url().should("include", "/form/quarterly-reports");
 
-    cy.findByRole("button", { name: /add another quarterly report/i }).click();
-    cy.findByRole("button", { name: /add another quarterly report/i }).click();
-    cy.findByRole("button", { name: /add another quarterly report/i }).click();
```

Comment:
Why do we no longer need to check quarterly report validation?

Reply:
Mistake - readded.